_activities/gravity.md

```diff
@@ -4,12 +4,22 @@ layout: activity
 # Content
 #
 title: "Gravity"
-title_fr: ""
+title_fr: "Gravité"
 age: 8-12
-objective: ""
-objective_fr: ""
-summary: ""
-summary_fr: ""
+objective: "Students will learn how to quantitatively describe how things fall under the affect of gravity. "
+objective_fr: "Les élèves vont apprendre à décrire quantitativement comment que les objets tombent sous l'éffet de la gravité."
+summary: "Students will drop a ball bearing into a small box that is filled with sand. This experiment will be performed twice; the first time the ball will have no initial velocity, and the second time the ball will have an initial velocity. Students will then use a flashlight to observe the craters made in the sand, and discuss the differences between the two craters."
+summary_fr: "Les élèves vont faire relacher une bille dans une petite boîte rempli de sable pour créer des cratères. Cette experience sera performée deux fois, la première fois la bille n'y aura pas de velocité initiale, et la deuxième fois il y aura une vélocité initiale. Les élèves se serviront ensuite d'une lampe de poche pour analyser la différence entre les deux cratère, et ils vont discutés leurs résultats. "
+downloads:
+  -
+	name: "Facilitator Guide - English"
```

Comment:
I'm not sure what these red boxes show - just double check that there are tabs and/or spaces here. I will also check after approving the pull request to make sure the links work.

Reply:
I double checked the spacing and I think it was okay, let me know if you notice something, sorry about that!

Reply:
No worries - great job with the edits!